roles/load_balancing/vars/ssl-vault.yml

```diff
@@ -1,1281 +0,0 @@
-$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Since this is an internal implementation detail, I figured we can take this hit (use this workaround) without compromising on an easy-to-use DSL for users of this repo.